source/DSCResources/DSC_Computer/DSC_Computer.psm1

```diff
@@ -92,18 +92,18 @@ function Get-TargetResource
     $convertToCimCredential = New-CimInstance `
         -ClassName DSC_Credential `
         -Property @{
-            Username = [System.String] $Credential.UserName
-            Password = [System.String] $null
-        } `
+        Username = [System.String] $Credential.UserName
+        Password = [System.String] $null
+    } `
         -Namespace root/microsoft/windows/desiredstateconfiguration `
         -ClientOnly
 
     $convertToCimUnjoinCredential = New-CimInstance `
         -ClassName DSC_Credential `
         -Property @{
-            Username = [System.String] $UnjoinCredential.UserName
-            Password = [System.String] $null
-        } `
```

Comment:
These changes are the vscode powershell extension so I don't really get why this is being reformatted.